src/OpenLoco/Graphics/ImageIds.h

```diff
@@ -270,6 +270,62 @@ namespace OpenLoco::ImageIds
     constexpr uint32_t fireball_28 = 3418;
     constexpr uint32_t fireball_29 = 3419;
     constexpr uint32_t fireball_30 = 3420;
+    constexpr uint32_t spark_00 = 3421;
+    constexpr uint32_t spark_01 = 3422;
+    constexpr uint32_t spark_02 = 3423;
+    constexpr uint32_t spark_03 = 3424;
+    constexpr uint32_t spark_04 = 3425;
+    constexpr uint32_t spark_05 = 3426;
+    constexpr uint32_t spark_06 = 3427;
+    constexpr uint32_t spark_07 = 3428;
+    constexpr uint32_t spark_08 = 3429;
+    constexpr uint32_t spark_09 = 3430;
+    constexpr uint32_t spark_10 = 3431;
+    constexpr uint32_t spark_11 = 3432;
+    constexpr uint32_t spark_12 = 3433;
+    constexpr uint32_t spark_13 = 3434;
+    constexpr uint32_t spark_14 = 3435;
+    constexpr uint32_t spark_15 = 3436;
+    constexpr uint32_t spark_16 = 3437;
+    constexpr uint32_t spark_17 = 3438;
+    constexpr uint32_t spark_18 = 3439;
+    constexpr uint32_t spark_19 = 3440;
+    constexpr uint32_t spark_20 = 3441;
+    constexpr uint32_t spark_21 = 3442;
+    constexpr uint32_t spark_22 = 3443;
+    constexpr uint32_t spark_23 = 3444;
+    constexpr uint32_t spark_24 = 3445;
+    constexpr uint32_t spark_25 = 3446;
+    constexpr uint32_t spark_26 = 3447;
+    constexpr uint32_t spark_27 = 3448;
+    constexpr uint32_t spark_28 = 3449;
+    constexpr uint32_t spark_29 = 3450;
+    constexpr uint32_t spark_30 = 3451;
+    constexpr uint32_t spark_31 = 3452;
+    constexpr uint32_t spark_32 = 3453;
+    constexpr uint32_t spark_33 = 3454;
+    constexpr uint32_t spark_34 = 3455;
+    constexpr uint32_t spark_35 = 3456;
+    constexpr uint32_t spark_36 = 3457;
+    constexpr uint32_t spark_37 = 3458;
+    constexpr uint32_t spark_38 = 3459;
+    constexpr uint32_t spark_39 = 3460;
+    constexpr uint32_t spark_40 = 3461;
+    constexpr uint32_t spark_41 = 3462;
+    constexpr uint32_t spark_42 = 3463;
+    constexpr uint32_t spark_43 = 3464;
```

Comment:
I get the impression that this (3449–3464) is a separate animation. Looking at the sprites in G1.dat, there is no transition from 3448 to 3449.

Could it be that these belong to another set of smoke sprites? It reminds me of the explosions in RCT2.

Reply:
Ok, I removed it.